pkg/util/paramtable/service_param.go

```diff
@@ -1094,15 +1095,23 @@ func (p *MinioConfig) Init(base *BaseTable) {
 	p.SecretAccessKey.Init(base.mgr)
 
 	p.UseSSL = ParamItem{
-		Key:          "minio.useSSL",
-		Version:      "2.0.0",
+		Key:          "minio.ssl.enabled",
+		Version:      "2.3.12",
 		DefaultValue: "false",
 		PanicIfEmpty: true,
 		Doc:          "Access to MinIO/S3 with SSL",
 		Export:       true,
 	}
 	p.UseSSL.Init(base.mgr)
```

Comment:
Add fallback key to be compatible with old instance

Reply:
FallbackKeys: []string{"minio.useSSL"},